skimage/filters/rank/generic.py

```diff
@@ -56,22 +55,59 @@
 
 from . import generic_cy
 
-
 __all__ = ['autolevel', 'bottomhat', 'equalize', 'gradient', 'maximum', 'mean',
            'geometric_mean', 'subtract_mean', 'median', 'minimum', 'modal',
            'enhance_contrast', 'pop', 'threshold', 'tophat', 'noise_filter',
            'entropy', 'otsu']
 
 
-def _handle_input(image, selem, out, mask, out_dtype=None, pixel_size=1):
+def _handle_input(image, selem=None, out=None, mask=None, out_dtype=None,
```

Comment:
Not sure how we originally came up with `_handle_input`; seems like a pretty terrible name! Can we be more descriptive?